pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -212,3 +250,14 @@ type DeploymentStorageStatus struct {
 	// PV name
 	PvName string `json:"pvName,omitempty"`
 }
+
+// TLSConfig extends the safe TLS configuration with file parameters.
+// +k8s:openapi-gen=true
+type TLSConfig struct {
+	// Path to the CA cert in the Prometheus container to use for the targets.
+	CAFile string `json:"caFile,omitempty"`
+	// Path to the client cert file in the Prometheus container for the targets.
+	CertFile string `json:"certFile,omitempty"`
+	// Path to the client key file in the Prometheus container for the targets.
+	KeyFile string `json:"keyFile,omitempty"`
```

Comment:
How about using a secret name here? as we did for other TLS configs.

Reply:
The current prometheus server does not support tls function. What about tls that we don't support in this pr?

Reply:
SGTM